### PR TITLE
data: arifaisal123

### PR DESCRIPTION
**Pull request by arifaisal123**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7811"></a>

### CODE REVIEW

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]
Hello! You have conflicts in your file. Please resolve and if you have any issues, feel free to ask here or in discord

**Comment by arifaisal123** [comment]
> Hello! You have conflicts in your file. Please resolve and if you have any issues, feel free to ask here or in discord

Thank you. Conflicts resolved for PR.

File: data/arifaisal123.json [hunk 1]
```diff
@@ -1,21 +1,55 @@
 {
   "name": "Arif Faisal",
-  "bio": "Full Stack Developer",
+  "bio": "I am a Full Stack Developer. Currently learning Django.",
+  "tags": ["Python", JavaScript", "Sass", "Bootstrap", "Flask"],
```
Comment (amandamartin-dev) [inline data/arifaisal123.json]:
```suggestion
  "tags": ["Python", "JavaScript", "Sass", "Bootstrap", "Flask"],
```
missing opening quotations so your json is failing, you can accept this edit if you like

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

**Comment by amandamartin-dev** [comment]
The conflict is resolved, but there is one small error in your json.

File: data/arifaisal123.json [hunk 2]
```diff
@@ -1,21 +1,55 @@
...
+      "url": "https://github.com/arifaisal123"
+    },
+    {
+      "icon": "FaYoutube",
+      "url": "https://youtube.com/@faisalcodes123"
+    },
+    {
+      "icon": "FaLink",
+      "url": "https://arifaisal123.github.io"
+    },
```
Comment (amandamartin-dev) [inline data/arifaisal123.json:21]:
```suggestion
    }
```
sorry, missed this one - extra comma here can be removed as well.

Comment (arifaisal123) [reply data/arifaisal123.json]: thank you, suggested change commited.

**Review by amandamartin-dev — APPROVED** [review]

File: data/arifaisal123.json (hunk 2 above)
Comment (amandamartin-dev) [reply data/arifaisal123.json]: awesome! merging now!

**Pull request merged**